core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -503,6 +503,8 @@ public void updateTask(TaskResult result) throws Exception {
 
         decide(workflowId);
 
+        workflowInstance = executionDAO.getWorkflow(workflowId);
```

Comment:
Keep in mind this could be a expensive call as by default it also loads all tasks, why is it needed here?

Reply:
This was leftover from debugging, I will remove this.